sqlite/analyze.c

```diff
@@ -1698,12 +1699,16 @@ static int analysisLoader(void *pData, int argc, char **argv, char **NotUsed){
     pIndex->bUnordered = 0;
     decodeIntArray((char*)z, nCol, aiRowEst, pIndex->aiRowLogEst, pIndex);
     if( pIndex->pPartIdxWhere==0 ) pTable->nRowLogEst = pIndex->aiRowLogEst[0];
-    /* COMDB2 MODIFICATION: assign noskipscan parameter */ 
-    pIndex->noSkipScan = is_comdb2_index_disableskipscan(pTable->zName, pIndex->zName);
+    /* COMDB2 MODIFICATION: assign noskipscan parameter, only if not disabled 
+    ** already.  */ 
+    if( !pIndex->noSkipScan ){
```

Comment:
Is there a way for this check to be if( ! remote table) instead? 
I want to have ability to enable/disable skipscan, and this line makes it stay disabled once disabled (unless restarting db); in regards to https://github.com/bloomberg/comdb2/pull/139.